src/main/react-front-end/src/views/Admin/HistoryComponent.js

```diff
@@ -286,7 +288,11 @@ class QueueComponent extends Component {
 		this.state.page=page
 		
 		this.setState({ page, response: this.state.response, responsesToDisplay: nextRecords});
+        var x = document.getElementsByClassName("rohit");

```

Comment:
Is this correct? Why is the class named Rohit?

@adas9542 @yinyifu

Reply:
rohit named this class rohit

Reply:
Oh.. it would be great if someone could refactor this and name it to something as per standards. 

Not urgent. Can be considered as the lowest priority task.